treesource.c

```diff
@@ -161,6 +163,36 @@ static void add_string_markers(struct property *prop)
 	}
 }
 
+void add_phandle_marker(struct dt_info *dti, struct property *prop, unsigned int offset)
+{
+	struct marker *m, **mi;
+	cell_t phandle;
+	struct node *refn;
+
+	m = xmalloc(sizeof(*m));
+	m->offset = offset;
+	m->type = REF_PHANDLE;
+
+	phandle = fdt32_ld((fdt32_t *)(prop->val.val + m->offset));
+	refn = get_node_by_phandle(dti->dt, phandle);
+
+	if (refn) {
+		if (refn->labels)
+			m->ref = refn->labels->label;
+		else
+			m->ref = refn->fullpath;
+	} else if (quiet < 1) {
+		fprintf(stderr, "Warning: node referenced by phandle 0x%x in property %s not found\n",
+			phandle, prop->name);
```

Comment:
I think you do need to consider `__fixups__` as well.  Without that, this error will be triggered if you're decompiling a dtbo with any external phandle references.